lib/lrama/grammar.rb

```diff
@@ -289,6 +293,18 @@ def append_special_symbols
       @accept_symbol = term
     end
 
+    def resolve_inline_rules
```

Comment:
Is this change intentional? It seems that this method is not called from anywhere, at least `bundle exec rake` successes even so deleting this method.

Reply:
sorry. I updated this PR.